### PR TITLE
Add mantid included conda environment recipe

### LINKED ISSUE #144: Add environment file with Mantid?
Our install docs refer to https://github.com/scipp/ess/blob/main/docs/environments/ess.yml for installation. Some users also want to use Mantid but may not know how to do this. I suggest we should therefore:

- Add a second environment file that includes mantid::mantid.
- Add a subsection or paragraph in the install docs, explaining how to install with Mantid.

### PR DESCRIPTION
**Pull request by YooSunYoung**
Fixes #144

### CODE REVIEW

File: docs/getting-started/installation.rst [hunk 1]
```diff
@@ -36,6 +36,16 @@ You may want to remove your old environment first, e.g.,
 and then proceed as per instructions above.
 The ``conda activate`` ensures that you are in your ``base`` environment.
 
+Mantid included environment file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to install `mantid` along with `ess`` environment, download :download:`ess-mantid.yml <../environments/ess-mantid.yml>` and run the commands below.
```
Comment (SimonHeybrock) [inline docs/getting-started/installation.rst]:
```suggestion
If you want to install ``mantid`` along with ``ess`` environment, download :download:`ess-mantid.yml <../environments/ess-mantid.yml>` and run the commands below.
```

**Review by SimonHeybrock — COMMENTED** [review]
I presume you have tested the instructions locally?

**Comment by YooSunYoung** [comment]
> 

Yes, I tried making a conda environment with these commands.
But I had to use mamba otherwise it was extremely slow. Should I maybe add the information in the instruction?

I just updated the rst file also. 
The preview didn't show me the problems in vs code... I will try to install another extension....

**Comment by SimonHeybrock** [comment]
> But I had to use mamba otherwise it was extremely slow. Should I maybe add the information in the instruction?

Yes, maybe we should just recommend `mamba` everywhere in these instructions?

File: docs/getting-started/installation.rst [hunk 2]
```diff
@@ -7,6 +7,19 @@ The easiest way to install ``ess`` is using `conda <https://docs.conda.io>`_.
 Packages from `Anaconda Cloud <https://conda.anaconda.org/scipp>`_ are available for Linux, macOS, and Windows.
 It is recommended to create an environment rather than installing individual packages.
 
+Mamba
+-------------
```
Comment (SimonHeybrock) [inline docs/getting-started/installation.rst]: Sphinx will complain if the length of the underline does not match the heading length.

**Comment by YooSunYoung** [comment]
> > But I had to use mamba otherwise it was extremely slow. Should I maybe add the information in the instruction?
> 
> Yes, maybe we should just recommend `mamba` everywhere in these instructions?

I just updated it to use `mamba` instead of `conda`.
I will update other installation documents after reviewing this one

**Comment by SimonHeybrock** [comment]
Should we change this everywhere @jl-wynen @nvaytet ?

**Comment by nvaytet** [comment]
I think we should indeed recommend the use of `mamba` everywhere :+1:

File: docs/getting-started/installation.rst [hunk 3]
```diff
@@ -7,6 +7,19 @@ The easiest way to install ``ess`` is using `conda <https://docs.conda.io>`_.
...
+
+Instead of ``conda``, we recommend ``mamba`` as a conda package manager.
+It is known that ``conda`` can be very slow and ``mamba`` can solve the problem.
+You can install ``mamba`` with the command below.
+
+   .. code-block:: sh
+      
+      conda install mamba -n base -c conda-forge
+
+Replace ``conda`` with ``mamba`` when you install any packages or create an environment.
```
Comment (nvaytet) [inline docs/getting-started/installation.rst]:
```suggestion
Once ``mamba`` is installed, you can replace ``conda`` with ``mamba`` when you install any packages or create an environment.
```

File: docs/getting-started/installation.rst [hunk 4]
```diff
@@ -7,6 +7,19 @@ The easiest way to install ``ess`` is using `conda <https://docs.conda.io>`_.
...
+
 With the provided environment file
```
Comment (nvaytet) [inline docs/getting-started/installation.rst]:
```suggestion
Use the provided environment file
```

File: docs/getting-started/installation.rst [hunk 5]
```diff
@@ -36,19 +49,36 @@ You may want to remove your old environment first, e.g.,
 and then proceed as per instructions above.
 The ``conda activate`` ensures that you are in your ``base`` environment.
 
+Mantid included environment file
```
Comment (nvaytet) [inline docs/getting-started/installation.rst]:
```suggestion
Using Mantid
```

File: docs/getting-started/installation.rst [hunk 6]
```diff
@@ -36,19 +49,36 @@ You may want to remove your old environment first, e.g.,
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to install ``mantid`` along with ``ess`` environment, download :download:`ess-mantid.yml <../environments/ess-mantid.yml>` and run the commands below.
```
Comment (nvaytet) [inline docs/getting-started/installation.rst]:
```suggestion
If you want to install ``mantid`` alongside ``ess`` in your environment, use the :download:`ess-mantid.yml <../environments/ess-mantid.yml>` environment file instead:
```

File: docs/getting-started/installation.rst [hunk 7]
```diff
@@ -36,19 +49,36 @@ You may want to remove your old environment first, e.g.,
...
 Without the provided environment file
 -------------------------------------
 
 To create a new conda environment with ``ess``:
 
 .. code-block:: sh
 
+   mamba create -n env_with_ess -c conda-forge -c scipp ess
+   # or
    conda create -n env_with_ess -c conda-forge -c scipp ess
```
Comment (nvaytet) [inline docs/getting-started/installation.rst]:
```suggestion
```

File: docs/getting-started/installation.rst [hunk 8]
```diff
@@ -36,19 +49,36 @@ You may want to remove your old environment first, e.g.,
...
 
 To add ``ess`` to an existing conda environment:
 
 .. code-block:: sh
 
+   mamba install -c conda-forge -c scipp ess
+   # or
    conda install -c conda-forge -c scipp ess
```
Comment (nvaytet) [inline docs/getting-started/installation.rst]:
```suggestion
```

**Review by nvaytet — COMMENTED** [review]
A few suggestions.

File: docs/getting-started/installation.rst [hunk 9]
```diff
@@ -7,7 +7,20 @@ The easiest way to install ``ess`` is using `conda <https://docs.conda.io>`_.
...
+You can install ``mamba`` with the command below.
+
+   .. code-block:: sh
+      
+      conda install mamba -n base -c conda-forge
+
+Once ``mamba`` is installed, you can replace ``conda`` with ``mamba`` when you install any packages or create an environment.
+
+Use the provided environment file
 ----------------------------------
```
Comment (nvaytet) [inline docs/getting-started/installation.rst]:
```suggestion
---------------------------------
```

File: docs/getting-started/installation.rst [hunk 10]
```diff
@@ -36,20 +49,33 @@ You may want to remove your old environment first, e.g.,
 and then proceed as per instructions above.
 The ``conda activate`` ensures that you are in your ``base`` environment.
 
+Using Mantid
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (nvaytet) [inline docs/getting-started/installation.rst]:
```suggestion
~~~~~~~~~~~~
```

**Review by nvaytet — COMMENTED** [review]
I think rst is picky and needs the lengths of underlines to agree with the title?

**Comment by YooSunYoung** [comment]
> I think rst is picky and needs the lengths of underlines to agree with the title?

Can we add this check in the `pre-commit`...?

**Comment by nvaytet** [comment]
> Can we add this check in the pre-commit...?

I'm not aware of a tool that does this, but if you find one that would be good to have, yes.

**Review by nvaytet — APPROVED** [review]

**Comment by YooSunYoung** [comment]
@SimonHeybrock Simmon, it says I am not authorized to merge this pr. Can you please merge it or give me the permit?

**Comment by nvaytet** [comment]
@YooSunYoung I think I give you permissions now

**Pull request merged**